package.json

```diff
@@ -14,7 +14,7 @@
     "posttest": "nsp check",
     "predist": "npm run fix",
     "prestart": "npm run fix",
-    "pretest": "npm run fix",
+    "pretest": "npm run fix && npm run dist",
```

Comment:
Not sure I agree. The requirement for the `test` script is that a server is running.